<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>System analysis 3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(173, 217, 250);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="9" column="4">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Polynomial degree</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="title">
         <string>Degree</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="text">
            <string>X1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="x3_deg">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="text">
            <string>X2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="x1_deg">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="x2_deg">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="locale">
            <locale language="Ukrainian" country="Ukraine"/>
           </property>
           <property name="text">
            <string>X3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="3" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="Ukrainian" country="Ukraine"/>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="select_output">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sample_spin">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>45</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="select_input">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="line_output">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="placeholderText">
         <string>Output file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="line_input">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QSpinBox" name="x2_dim">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>X1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="x1_dim">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>X2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>X3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="x3_dim">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="y_dim">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Control panel</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="plot_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(155, 194, 224);</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(155, 194, 224);</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Solve</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Weights</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="weights_box">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="currentText">
         <string>Average</string>
        </property>
        <item>
         <property name="text">
          <string>Average</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Scaled</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QCheckBox" name="lambda_check">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="Ukrainian" country="Ukraine"/>
     </property>
     <property name="text">
      <string>Use 3-block lambda</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="5">
    <widget class="QTextBrowser" name="results_field">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255); color: black;</string>
     </property>
     <property name="locale">
      <locale language="Ukrainian" country="Ukraine"/>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.Menio Regular'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Results</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 217, 250); color: black;</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="title">
      <string>Polynomial type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radio_cheb_t">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Чебышев I</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_sh_cheb_t">
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Чебышев I смещенный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_sh_cheb_u">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="locale">
         <locale language="Ukrainian" country="Ukraine"/>
        </property>
        <property name="text">
         <string>Чебышев II смещенный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="custom_check">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_cheb_t</tabstop>
  <tabstop>radio_sh_cheb_t</tabstop>
  <tabstop>x1_deg</tabstop>
  <tabstop>x2_deg</tabstop>
  <tabstop>x3_deg</tabstop>
  <tabstop>x1_dim</tabstop>
  <tabstop>x2_dim</tabstop>
  <tabstop>sample_spin</tabstop>
  <tabstop>line_input</tabstop>
  <tabstop>select_input</tabstop>
  <tabstop>line_output</tabstop>
  <tabstop>select_output</tabstop>
  <tabstop>exec_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exec_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>exec_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plot_button</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>plot_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x1_dim</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>dimension_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x2_dim</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>dimension_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x3_dim</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>dimension_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x1_deg</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>degree_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x2_deg</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>degree_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x3_deg</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>degree_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_cheb_t</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>type_modified(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_sh_cheb_t</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>type_modified(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lambda_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>Form</receiver>
   <slot>lambda_calc_method_changed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weights_box</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>weights_modified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_dim</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>dimension_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sample_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>samples_modified(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_input</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>input_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_output</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>output_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Form</sender>
   <signal>output_changed(QString)</signal>
   <receiver>line_output</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Form</sender>
   <signal>input_changed(QString)</signal>
   <receiver>line_input</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_input</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>input_modified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_output</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>output_modified(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>input_changed(QString)</signal>
  <signal>output_changed(QString)</signal>
  <signal>x1_dim_changed(int)</signal>
  <signal>x2_dim_changed(int)</signal>
  <signal>x3_dim_changed(int)</signal>
  <signal>x1_deg_changed(int)</signal>
  <signal>x2_deg_changed(int)</signal>
  <signal>x3_deg_changed(int)</signal>
  <signal>type_cheb()</signal>
  <signal>type_lege()</signal>
  <signal>type_lagg()</signal>
  <signal>type_herm()</signal>
  <slot>input_clicked()</slot>
  <slot>output_clicked()</slot>
  <slot>samples_modified(int)</slot>
  <slot>dimension_modified(int)</slot>
  <slot>degree_modified(int)</slot>
  <slot>type_modified(bool)</slot>
  <slot>plot_clicked()</slot>
  <slot>exec_clicked()</slot>
  <slot>input_modified(QString)</slot>
  <slot>output_modified(QString)</slot>
  <slot>weights_modified(QString)</slot>
  <slot>lambda_calc_method_changed(bool)</slot>
 </slots>
</ui>
